test/e2e/node/runtimeclass.go

```diff
@@ -143,18 +138,61 @@ var _ = SIGDescribe("RuntimeClass [Feature:RuntimeClass]", func() {
 	// TODO(tallclair): Test an actual configured non-default runtimeHandler.
 })
 
+// createCRD creates CRD for RuntimeClass.
+func createCRD(f *framework.Framework) {
+	crdFilePath := "cluster/addons/runtimeclass/runtimeclass_crd.yaml"
+	data := testfiles.ReadOrDie(crdFilePath, Fail)
```

Comment:
It's a lot simpler to just use kubectl to load the manifest, and I think that should be acceptable for the test. Something like:
```
_, err := framework.RunKubectl("apply", "-f", crdFilePath)
```

Reply:
`framework.RunKubectl("apply", "-f", crdFilePath)` may not work, because `crdFilePath` is a test file, and requires special care to be read: https://github.com/kubernetes/kubernetes/blob/5b6a23f3af47ad746f50d89d69941a9c48de7a48/test/e2e/framework/testfiles/testfiles.go#L77